recipe/meta.yaml

```diff
@@ -9,14 +9,18 @@ source:
   url: https://static.rust-lang.org/dist/2020-11-24/rust-nightly-powerpc64le-unknown-linux-gnu.tar.gz  # [ppc64le]
 
 build:
-  number: 2
+  number: 3
   # the distributed binaries are already relocatable
   binary_relocation: False
-
+  ignore_run_exports:
```

Comment:
We don't need `ignore_run_exports` in this one since it was not used as build time.